cfstore/cfstore_client.go

```diff
@@ -1,46 +1,14 @@
 package cfstore
```

Comment:
Does this file provide any value over the code just using go-cfclient directly now? Could we just remove it?

Reply:
Unfortunately I don't think the go-cfclient library offers an interface for the client that we can create a fake version of. Being able to fake this client provides a convenient and idiomatic way of testing when certain responses come back from the CF API.

Reply:
It's become an emerging idiom in this sort of context to declare the interface at the point that it's consumed. As things stand, there shouldn't be a need for the `Client` type because `*cfclient.Client` itself already implements the defined interface.

This means that the consuming code can define an interface that it accepts, and we can then pass in either an instance of `*cfclient.Client`, or a fake that also implements the interface.